Docs build broken

```
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.local/lib/python3.12/site-packages/oca_repo_maintainer/cli/pages.py", line 22, in pages
    GHPageGenerator(conf_dir, org, path).run()
  File "/home/runner/.local/lib/python3.12/site-packages/oca_repo_maintainer/tools/gh_pages.py", line 33, in run
    self._generate_repo_pages()
  File "/home/runner/.local/lib/python3.12/site-packages/oca_repo_maintainer/tools/gh_pages.py", line 67, in _generate_repo_pages
    team = self.conf_psc[team_slug]["name"]
           ~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'local-finland-maintainers-project-leader'
Error: Process completed with exit code 1.
```

--- a/conf/psc/l10n.yml
+++ b/conf/psc/l10n.yml
@@ -202,11 +202,6 @@ local-germany-maintainers:
     - tv-openbig
   name: Local germany maintainers
   representatives: []
-local-germany-maintainers-project-leader:
-  members:
-    - tv-openbig
-  name: Local germany maintainers project leader
-  representatives: []
 local-germany-maintainers-psc-representative:
   members:
     - tv-openbig

--- a/conf/repo/l10n.yml
+++ b/conf/repo/l10n.yml
@@ -322,7 +322,7 @@ l10n-germany:
   category: L10n
   default_branch: "18.0"
   name: L10n Germany
-  psc: local-germany-maintainers-project-leader
+  psc: local-germany-maintainers
   psc_rep: local-germany-maintainers-psc-representative
 l10n-greece:
   branches:

--- a/conf/repo/l10n.yml
+++ b/conf/repo/l10n.yml
@@ -284,7 +284,7 @@ l10n-finland:
   category: L10n
   default_branch: "18.0"
   name: L10n Finland
-  psc: local-finland-maintainers-project-leader
+  psc: local-finland-maintainers
   psc_rep: local-finland-maintainers-psc-representative
 l10n-france:
   branches:

